includes/class-scd-ext-drip-email.php

```diff
@@ -263,7 +263,7 @@ public function send_bulk_drip_notifications( $users_lessons ) {
 				 *
 				 * @param string $email_heading
 				 */
-				'heading'   => apply_filters( 'scd_email_heading', __( 'Content Drip', 'sensei-content-drip' ) ),
+				'heading'   => apply_filters( 'scd_email_heading', __( 'New Content Available', 'sensei-content-drip' ) ),
```

Comment:
Maybe "New Lessons Available" since we're using "lessons" everywhere else in the email, plus the content really is just lessons.

Reply:
Done! Thanks.